reference-service/src/main/resources/config/application-stage.yml

```diff
@@ -6,7 +6,6 @@ spring:
   flyway:
     password: ${DBPASSWORD}
     schemas: ${DBNAME}
-    locations: classpath:db/migration/common,classpath:db/migration/tenant/hee,classpath:db/migration/stage
```

Comment:
I think we still need the other locations, we just need to drop the unused `,classpath:db/migration/stage`

Reply:
This whole line was added when we were handling DBC changes: https://github.com/Health-Education-England/TIS-REFERENCE/pull/286

Reply:
Yep, I wasn't thinking about the fallback to default props.  Thanks for the additional pointer!

Reply:
Thank you Pepe.